src/resources/users/UserForm.tsx

```diff
@@ -41,11 +42,24 @@ export default function UserForm({ isEdit }: FormProps): React.ReactElement {
     password: '',
     role: 'rco-user'
   }
+  const style = {
+    position: 'absolute',
+    top: '20%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 600,
+    textAlign: 'center',
+    color: 'gray',
+    border: '3px solid silver',
+    p: 4,
+    borderRadius: 2
+  }
```

Comment:
I have tested that, and I would say it does not be better. Because when I use `?? false` at the end of a line, which causing error. The error `Unexpected constant nullishness on the left-hand side of a ?? expression` is due to TypeScript's strict nullish checks. The nullish coalescing operator ?? is intended to be used with expressions that might be null or undefined. However, the optional chaining (?.) in the left-hand side of this expression already handles null or undefined values.
To resolve this, we can use a simpler and more conventional approach: `const isPowerUser = record?.role === 'rco-power-user';` 
This accomplishes the same logic without triggering the TypeScript error. If record?.role is 'rco-power-user', isPowerUser will be true. If record?.role is null or undefined, isPowerUser will be undefined. If you need isPowerUser to always be a boolean (true or false), we can keep the ?? false, but it might be unnecessary based on the context.